surfer/viz.py

```diff
@@ -1885,6 +1886,21 @@ def set_surf(self, surf):
             if brain._f.scene is not None:
                 brain._f.scene.reset_zoom()
 
+    @property
+    def _brain_color(self):
+        geo_actor = self._brain_list[0]['brain']._geo_surf.actor
+        if self._brain_list[0]['brain']._using_lut:
+            bgcolor = np.mean(
+                self._brain_list[0]['brain']._geo_surf.module_manager
+                .scalar_lut_manager.lut.table.to_array(), axis=0)
+        else:
+            bgcolor = geo_actor.property.color
+            if len(bgcolor) == 3:
+                bgcolor = bgcolor + (1,)
+            bgcolor = 255 * np.array(bgcolor)
+        bgcolor[-1] *= geo_actor.property.opacity
+        return bgcolor
```

Comment:
I don't like to see properties that do actual work although I agree here it's unlikely to be called in a loop.

Reply:
This seems roughly consistent with what is already done in PySurfer -- it's mostly about triaging values rather than doing any big calculations